README.adoc

```diff
@@ -231,14 +241,15 @@ InventoryClient.java
 include::finish/query/src/main/java/io/openliberty/guides/query/client/InventoryClient.java[]
 ----
 
-The return types of the [hotspot=getSystem file=1]`getSystem()` method are now an `Observable` object instead of a `CompletionStage` interface. 
-http://reactivex.io/RxJava/javadoc/io/reactivex/Observable.html[Observable^] is a collection of data that awaits to be subscribed to before it can release any data and is part of RxJava. 
-The [hotspot=rx file=1]`rx()` method now needs to contain `RxObservableInvoker.class` as an argument. This is to call the specific invoker, `RxObservableInvoker`, for the Observable class provided by Jersey. 
+The return type of the [hotspot=getSystem file=1]`getSystem()` method is now an `Observable` object instead of a `CompletionStage` interface. 
+http://reactivex.io/RxJava/javadoc/io/reactivex/Observable.html[Observable^] is a collection of data that waits to be subscribed to before it can release any data and is part of RxJava. 
+The [hotspot=rx file=1]`rx()` method now needs to contain `RxObservableInvoker.class` as an argument.
+This argument calls the specific invoker, `RxObservableInvoker`, for the `Observable` class that's provided by Jersey. 
 In the [hotspot=getSystem file=1]`getSystem()` method, the [hotspot=register file=1]`register(RxObservableInvokerProvider)` method call registers the `RxObservableInvoker` class, which means that the client can recognize the invoker provider.
 
-Sometimes there are scenarios where a producer will generate more data than the consumers can handle. 
-In cases like these, JAX-RS can deal with this issue using RxJava's `Flowable` with `backpressure`. 
-You can learn more with this post about https://openliberty.io/blog/2019/04/10/jaxrs-reactive-extensions.html[JAX-RS reactive extensions with RxJava Backpressure^]
+In some scenarios, a producer might generate more data than the consumers can handle. 
+JAX-RS can deal with cases like these by using the RxJava `Flowable` class with backpressure. 
+To learn more about RxJava and backpressure, see https://openliberty.io/blog/2019/04/10/jaxrs-reactive-extensions.html[JAX-RS reactive extensions with RxJava Backpressure^]
```

Comment:
Add a period after this sentence and don't capitalize the second instance of "backpressure".